crates/holochain_cascade/src/lib.rs

```diff
@@ -623,27 +624,25 @@ where
         options: GetOptions,
     ) -> CascadeResult<Option<EntryDetails>> {
         let authoring = self.am_i_authoring(&entry_hash.clone().into())?;
-        let authority = self.am_i_an_authority(entry_hash.clone().into()).await?;
+        let authority = authoring || self.am_i_an_authority(entry_hash.clone().into()).await?;
```

Comment:
I find this misleading, semantically, and suggest to rename the variable to something like `should_not_fetch` or `search_cache_only`. Or another more suitable identifier.